docs/verify/simulators.mdx

```diff
@@ -38,7 +38,13 @@ Simulates a quantum circuit by computing the wavefunction of the qubit’s state
 'multiplexer', 'initialize', 'kraus', 'roerror', 'delay']
 ```
 
-## Stabilizer
+**Code example**
+
+```python
+backend = service.get_backend("simulator_statevector")
```

Comment:
I think these code snippets should also include the imports and initialising the runtime service

Reply:
Per https://github.com/Qiskit/documentation/issues/169, we also want files with code blocks to be Jupyter notebooks instead so that we can verify the code still works properly.

(From my perspective, I'm fine deferring that migration to Jupyter for this file, as long as its tracked and done in a follow up)